manifests/remote.pp

```diff
@@ -6,15 +6,15 @@
   $enable            = true,
   $interface         = ['::1', '127.0.0.1'],
   $port              = 953,
-  $server_key_file   = "${unbound::params::confdir}/unbound_server.key",
-  $server_cert_file  = "${unbound::params::confdir}/unbound_server.pem",
-  $control_key_file  = "${$unbound::params::confdir}/unbound_control.key",
-  $control_cert_file = "${$unbound::params::confdir}/unbound_control.pem",
-  $group             = $unbound::params::group,
-  $confdir           = $unbound::params::confdir,
-) inherits unbound::params {
+  $server_key_file   = "${unbound::confdir}/unbound_server.key",
+  $server_cert_file  = "${unbound::confdir}/unbound_server.pem",
+  $control_key_file  = "${unbound::confdir}/unbound_control.key",
+  $control_cert_file = "${unbound::confdir}/unbound_control.pem",
+  $group             = $unbound::group,
+  $confdir           = $unbound::confdir,
+) inherits unbound {
```

Comment:
I'd agree with @igalic here.  Inheritance isn't really recommended, and just about the only place its deemed acceptable is on the params class.  Mostly just stylistic.